vector/src/test/java/im/vector/app/features/crypto/quads/SharedSecureStorageViewModelTest.kt

```diff
@@ -44,6 +46,7 @@ class SharedSecureStorageViewModelTest {
 
     private val stringProvider = FakeStringProvider()
     private val fakeSession = FakeSession()
+    private val matrix: Matrix = mockk()
```

Comment:
Maybe we could create a `FakeMatrix` class to encapsulate the mock? To mutualise the mock for future tests.

Reply:
There is actually a `TestMatrix` class that kind of does this, but it's a completely separate class, not related by inheritance to Matrix. Since Matrix is a closed class, I can only think about either making it open or using an interface for this. Or maybe injecting only `SecureStorageService` here instead of the whole `Matrix` class only to retrieve one of its dependencies.

Reply:
I didn't know there was a TestMatrix class. No worry, I was just thinking about something like that:
```
class FakeMatrix() {

    val instance = mockk<Matrix>()

}
```
No need to implement interface since as you said `Matrix` is a class.